preprocessor.py

```diff
@@ -16,32 +16,49 @@ def __init__(self):
                     'MONGODB_DB': "ecosystem_mapping",
                     'MONGODB_FILTERED_COLLECTION': "filtered_collection",
                     'MONGODB_HTML_COLLECTION': "html_collection",
+                    'MONGODB_LINK_COLLECTION': "link_collection",
                     'MONGODB_TEXT_COLLECTION': "text_collection"}
         connection = MongoClient(
             settings['MONGODB_SERVER'],
             settings['MONGODB_PORT']
         )
         self.db = connection[settings['MONGODB_DB']]
-        self.html_collection = self.db[settings['MONGODB_HTML_COLLECTION']]
         self.filtered_collection = self.db[settings['MONGODB_FILTERED_COLLECTION']]
+        self.html_collection = self.db[settings['MONGODB_HTML_COLLECTION']]
+        self.link_collection = self.db[settings['MONGODB_LINK_COLLECTION']]
 
     def load_save(self):
         """Loads in from MongoDB and saves to MySQL."""
         base_urls = self.html_collection.distinct("base_url")
         for base_url in base_urls:
-            print("Cleaning input from URL: %s" % base_url)
-            tier = float('inf')
+            print("Processing URL: %s" % base_url)
             for data in self.html_collection.find({"base_url": base_url}):
+
+                external_links = data['dst_url']
```

Comment:
This is the part where it needs to actually extract the external links from the HTML body we have since we don't want to rerun scrapy